server/src/test-data.js

```diff
@@ -86,15 +86,32 @@ export const loadTestData = () =>
               name: 'VR Wollongong Escarpment',
               details: 'VR operators required to assist with rescue',
               group,
-            }).then(event =>
-              Creators.eventResponse({
-                status: 'On Route',
-                detail: 'Attending',
-                destination: 'Scene',
-                eta: 1517443200,
-                event,
-                user,
-              }),
+            }).then(event => Promise.all([
+                Creators.eventResponse({
+                  status: 'Responding',
+                  detail: 'ETA 10 min',
+                  destination: 'Scene',
+                  eta: 1517443200,
+                  event,
+                  user,
+                }),
+                Creators.eventResponse({
+                  status: 'Responding',
+                  detail: 'WOL39',
+                  destination: 'Scene',
```

Comment:
down the road I think we should have defined somewhere the options for destination. might be that groups have meeting places, or organisations have meeting places.

Reply:
agreed